postgis/postgis.sql.in

```diff
@@ -3479,9 +3479,16 @@ CREATE OR REPLACE FUNCTION ST_OffsetCurve(line geometry, distance float8, params
 CREATE OR REPLACE FUNCTION ST_GeneratePoints(area geometry, npoints numeric)
        RETURNS geometry
        AS 'MODULE_PATHNAME','ST_GeneratePoints'
-       LANGUAGE 'c' IMMUTABLE STRICT _PARALLEL
+       LANGUAGE 'c' VOLATILE STRICT _PARALLEL_RESTRICTED
```

Comment:
@Komzpa What do you think about the change from  to `VOLATILE`? IRC you set this up as `IMMUTABLE` on purpose (for performance).

Reply:
Without a seed, this should return a random multipoint each time. (Note that `random()` is VOLATILE STRICT). The IMMUTABLE version with a seed parameter is below.

Reply:
does it have to be parallel restricted? if it's volatile and resets seed on start why not start it in parallel worker too?